index.html

```diff
@@ -177,11 +177,15 @@ <h2 class="section-title">Anchor placement</h2>
 				</div>
 
 				<div class="section-group">
-					<h3>Install using bower</h3>
+					<h3>Install using Yarn, Npm, Bower</h3>
 
+					<pre><code>yarn add aos</code></pre>
+					
+					<pre><code>npm install aos -S</code></pre>
```

Comment:
For consistency I'd use `--save` flag here as well. New npm saves packages by default, unfortunately we can't assume everyone has new node version installed ;/